src/rendering/src/pages/_app.tsx

```diff
@@ -62,6 +64,15 @@ function App({ Component, pageProps, router }: AppPropsWithLayout): JSX.Element
       PageController.getContext().setLocaleCountry('us');
     }
   }, []);
+
+  const pageUri = usePathname();
+  useEffect(() => {
+    if (isSearchSDKEnabled) {
+      PageController.getContext().setPageUri(pageUri);
+      trackPageViewEvent('page');
```

Comment:
This should probably be `trackEntityPageViewEvent('content', { items: [{ id: pageUri }] });` for results to appear in CEC.